marketplace-ui/src/app/modules/product/product-detail/product-detail.component.html

```diff
@@ -20,13 +20,16 @@
           <div class="d-flex">
             <div class="logo__image-container">
               <img
+                width="70"
```

Comment:
duplicate with logo__image, why we need to add it?

Reply:
because when we use ngSrc, it requires width and height or fill attributes.